RNA_pipeline/rna_strand.wdl

```diff
@@ -0,0 +1,67 @@
+version 1.0
+
+
+workflow RNAStrandWorkflow {
+    input {
+        File input_bam
+        File refseq = "gs://cds-vep-data/ucsc_ncbi_refseq_curated_hg38_20240303.bed"
+
+        String sample_id
+        String docker_image = "us-docker.pkg.dev/depmap-omics/public/strand:24q2"
+    }
+
+    call strand_check_task as strand {
+        input:
+            input_bam=input_bam,
+            refseq=refseq,
+            sample_id=sample_id,
+            docker_image=docker_image,
+    }
+
+    output {
+        Float percentage_1pp1mm = strand.line_1pp1mm
+        Float percentage_1pm1mp = strand.line_1pm1mp
+        # If fraction explained by 1+-,1-+,2++,2-- is higher than 0.7, it is stranded 
+        Boolean inferred_stranded = percentage_1pm1mp > 0.7
+    }
+}
+
+
+# Standard interface to run vcf to maf
+task strand_check_task {
+    input {
+        File input_bam
+        File refseq
+        String sample_id
+
+        String docker_image
+        Int preemptible=2
+        Int boot_disk_size=60
+        Int cpu = 10
+        Int mem = 80
+        Int additional_disk_gb = 0
+    }
+
+    Int disk_space = ceil(size(input_bam, "GiB")) + 10 + additional_disk_gb
```

Comment:
What does "10" refer to? It looks like a magic number here

Reply:
Boilerplate: A standard amount of additional disk space the command can use for temp files and the like.